v2/emailpassword/troubleshooting/cors-issues.mdx

```diff
@@ -188,6 +188,35 @@ app.use(
 );
 ```
 
+<br />
+
+### `[Error] from origin 'http://localhost:3000' has been blocked by CORS policy ... the value of 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's crednetial mode is 'include'`
+
+This is a pretty long error message but there are two key takeaways here. 
+
+First, similar to our previous issue, this appears when `Access-Control-Allow-Origin` header includes `*`. To fix, add in the correct origin value, http://localhost:3000 in this case, in the CORS settings.
+
+Second, for services that come back with a different origin URL each time like Google auth, we'd want to use the wildcard in the following manner - `https://*.domain.com` . 
```

Comment:
I don't understand this, which means probably no one will. Can you be more clear about this? Or just remove this second point if you aren't sure about it

Reply:
Hmm, ok I can remove the second point.